styles.css

```diff
@@ -0,0 +1,141 @@
+
+body {
+  background: url(images/background.jpg);
+  background-size: cover;
+
```

Comment:
Nice use of the background rules for styling.  Check out this article (https://css-tricks.com/perfect-full-page-background-image/).